packages/ubus-lime-fbw/files/www/cgi-bin/hostname

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+echo "Status: 200"
+echo "Content-type: text/plain;"
+echo "Cache-control: max-age=0, no-cache;"
+echo "Access-Control-Allow-Origin: *"
+echo "Access-Control-Allow-Methods: GET;"
+echo "Access-Control-Allow-Headers: X-PINGOTHER, Content-Type;"
+echo
+echo $HOSTNAME
```

Comment:
missing new line at the end of the file

Reply:
fixed in 42b9ba9c7cdc476f4e4c890b606adda409aed1ab